.github/workflows/test.yml

```diff
@@ -0,0 +1,27 @@
+name: Update the Code Coverage
+
+on:
+  workflow_dispatch
+
+jobs:
+  coverage:
```

Comment:
The job should be renamed to publish-code-coverage-report.

Reply:
This job is also renamed to `publish-code-coverage-report`